lib/package.json

```diff
@@ -81,7 +82,8 @@
     "classnames": "^2.2.6",
     "fuse.js": "^6.4.6",
     "prop-types": "^15.7.2",
-    "ramda": "^0.27.1"
+    "ramda": "^0.27.1",
+    "react-vis": "^1.11.7"
```

Comment:
Do we want this to be a lib dependency? It doesn't seem like it really needs to be

Reply:
> Do we want this to be a lib dependency? It doesn't seem like it really needs to be

I thought it would be convenient for users to just be able to use the library without having to install it themselves. Should I just remove it?

Reply:
> Should I just remove it?

I can see arguments either way.  I'll ask around in the JavaScript slack channel to see what would make more sense to people.

Reply:
> I can see arguments either way. I'll ask around in the JavaScript slack channel to see what would make more sense to people.

Actually on second thought I've decided against it.  IMO any codebase which references a given module in one of its `import` statements should always declare a direct dependency on that module.  So following that rule, downstream projects which use react-vis should always have their own explicit dependency on it, so there isn't really anything gained by having the RSC lib also declare a dependency on it.